data/getdata.sh

```diff
@@ -7,8 +7,8 @@ echo "Unzipping the data files (overwrites any existing .tsv files)..."
 gunzip -vf *.gz
```

Comment:
(GH isn't letting me comment on line 4, so doing here instead)

TIOLI: To Manoj's point about error handling, should we error check on the curl call? Literal question, because it's not likely, but possible, that the domain changes, the files are moved, the network is down, or the site is unavailable for some other reason.